workers/data_refinery_workers/downloaders/array_express.py

```diff
@@ -76,11 +73,9 @@ def _extract_file(batches: List[Batch], job_id: int) -> None:
 @shared_task
 def download_array_express(job_id: int) -> None:
     job = utils.start_job(job_id)
+    batches = list(job.batches.all())
```

Comment:
You don't have to convert the QuerySet `job.batches.all()` into a list to get its size and the first element. You can use `count()` and `first` methods instead.